polaris-react/src/components/KeyboardKey/KeyboardKey.tsx

```diff
@@ -1,15 +1,20 @@
 import React from 'react';
 
+import {classNames} from '../../utilities/css';
+
 import styles from './KeyboardKey.scss';
 
+type Size = 'small';
+
 export interface KeyboardKeyProps {
-  /** The content to display inside the key */
   children?: string;
+  size?: Size;
 }
-
-export function KeyboardKey({children}: KeyboardKeyProps) {
+export function KeyboardKey({children, size}: KeyboardKeyProps) {
```

Comment:
```suggestion
export function KeyboardKey({children = '', size}: KeyboardKeyProps) {
```
This will simplify the code below

Reply:
Thank you! I was also considering making `children` required but figured to keep it as is for now 👌